### PR TITLE
Add Amplitude feature setUserProperties

### PR DESCRIPTION
**Pull request by chasedut**
Looking to add support for setting Amplitude user properties, which is present in the initialize function

see initialize here: 
screen shot 2018-01-26 at 12 07 43 pm

docs below on setUserProperties:

https://amplitude.zendesk.com/hc/en-us/articles/207108327-Step-4-Set-User-Properties-and-Event-Properties

https://amplitude.zendesk.com/hc/en-us/articles/115001361248-JavaScript-SDK-Installation#setting-multiple-user-properties

### CODE REVIEW

**Review by timurista — APPROVED** [review]
Nice, LGTM!

File: src/index.js [hunk 1]
```diff
@@ -85,7 +85,22 @@ const Amplitude = {
...
     amplitude.getInstance().setUserId(userID);
+  },
+
+  /**
+   * setUserProperties:
+   *
+   * @userProps userProps {Object} required
+   */
+  setUserProperties: function (userProps) {
+    console.log('amplitude userProps...');
```
Comment (boxman0617) [inline src/index.js]: Can you wrap this in `_debug` and use `log()` instead? Look at some of the examples above in method: `amplitude`

**Review by boxman0617 — CHANGES_REQUESTED** [review]
Just one quick request.

File: src/index.js (hunk 1 above)
Comment (boxman0617) [reply src/index.js]: or remove it?

**Comment by rorygarand** [comment]
@chasedut @timurista @boxman0617 thanks!

i have some other changes i need to get merged in today, but this will go in right afterwards.

**Review by boxman0617 — APPROVED** [review]
LGTM!

**Review by rorygarand — APPROVED** [review]

**Pull request merged**